c2corg_ui/static/js/constants.js

```diff
@@ -37,5 +37,6 @@ app.constants = {
     FORM_PROJ: 'EPSG:4326',
     DATA_PROJ: 'EPSG:3857'
   },
+  discourseURL: 'http://c2corgv6-discourse.demo-camptocamp.com/',
```

Comment:
You cannot hardcode URLs in the JS files (ditto for ``imagesURL`` below) because you cannot change those URLs without making a new commit. You have to use a ``module.value`` and inject it in your JS controllers. See for instance ``apiUrl`` https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/base.html#L143